data/campaigns/Eastern_Invasion/scenarios/14_The_Drowned_Plains.cfg

```diff
@@ -1868,10 +1909,12 @@ I offer one final prayer to the Light, that at least my family may be spared and
 
         [message]
             speaker=second_unit
+            #po: speaker is an undead horseman, who's fighting Gaennell. Unclear whether this is to a memory, or because Gaennell is a Shadow Mage
             message= _ "... Necromancer... you are a blight..."
         [/message]
         [message]
             speaker=Gaennell
+            #po: to an undead horseman in an [event]name=attack
```

Comment:
Using WML in a translator hint seems like a bad idea. Can't you just say "while attacking" or "while being attacked" or something like that?